playbooks/infra/deploy-bm-hypervisor.yml

```diff
@@ -0,0 +1,227 @@
+---
+- name: Create a Kickstart-enabled ISO
+  hosts: bastion
+  gather_facts: true
+  vars:
+    iso_mount_path: "{{ dest_iso_dir }}/mount"
+    os_install_path: "{{ dest_iso_dir }}/os-install"
+    location: rdu
+    system_iso_link: "{%- if location == 'rdu' -%} {{ system_iso_rdu_link }} {%- else -%} {{ system_iso_tlv_link }} {%- endif -%}"
+  tasks:
+    - name: Set ISO name
+      ansible.builtin.set_fact:
+        iso_name: installation.iso
+
+    - name: Prepare kickstart iso
+      ansible.builtin.import_role:
+        name: kickstart_iso
+      vars:
+        kickstart_iso_link: "{{ system_iso_link }}"
+        kickstart_iso_name: "{{ iso_name }}"
+        kickstart_iso_file_desire_location: /opt/http_store/data
+        kickstart_iso_timezone: "{{ hostvars['hypervisor'].timezone }}"
+        kickstart_iso_password: "{{ ansible_password }}"
+        kickstart_iso_username: "{{ ansible_user }}"
+        kickstart_iso_net_config:
```

Comment:
Maybe we can use something like
`kickstart_iso_net_config: "{{ hostvars['hypervisor'] }}"`

Reply:
Fixed.